bokehjs/src/coffee/renderer/annotation/box_annotation.coffee

```diff
@@ -50,6 +50,7 @@ class BoxAnnotation extends HasParent
 
   defaults: ->
     return _.extend {}, super(), {
+      plot: null
```

Comment:
This should have been added to the super class. If there is none, then you should add one.